mac

```diff
@@ -33,7 +33,7 @@ log_info() {
 
 if ! command -v brew &>/dev/null; then
   log_info "Installing Homebrew, a good OS X package manager ..."
-    ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+    ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh)"
```

Comment:
Looking at https://brew.sh it should probably be:

```suggestion
  /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
```

(feeding a `sh` script to ruby probably won't work 😛)

Reply:
That is a fair point :-) I probably got overzealous

In my defence, I just fixed the obvious 404
Don't think this part of the code really gets tested all that much?

Reply:
it's only "tested" by people who use it 😄 (and then it's probably just once 😉)